src/content/docs/en/user-guide/setup.mdx

```diff
@@ -31,6 +32,8 @@ The **Add to wallet** links may not be compatible with every wallet. If you are
 | Currency Symbol    | ETH                                                              | ETH                                                                          |
 | Block Explorer URL | [https://sepolia.scrollscan.com](https://sepolia.scrollscan.com) | [https://sepolia.etherscan.io](https://sepolia.etherscan.io)                 |
 
+<RPCTable />
```

Comment:
Example usage

**please put the table in the appropriate place or delete this before merge**